app/templates/application.hbs

```diff
@@ -30,7 +30,9 @@
   {{#banner.nav}}
     <ul class="menu vertical large-horizontal">
       <li>{{link-to 'Search' 'show-geography'}}</li>
-      {{sign-in}}
+      {{#if (in-array (array "development" "test") this.ENV.environment)}}
```

Comment:
@trbmcginnis @godfreyyeung  Just FYI this will never show up in any production environment, so if we wanted to deploy this to a separate staging environment (ember s -e production) it wouldn't show any of these features. You could force a development deploy but that is weird.

What I meant by a "feature flag" was that the environment.js, which you have (`this.ENV.environment`), would look for an environmental variable (`ENABLE_LUPP_FEATURES=true ember s`).

At this point it doesn't quite matter but it will when we actually need to deploy these features to a staging environemnt....

Reply:
Ohh I see. I was thinking we would just remove these conditionals when the sign-in is no longer "Development" ... but yeah this is not as good as the feature flag solution you mentioned.

Reply:
@godfreyyeung no, don't close, please fix lint things and merge

Reply:
@godfreyyeung This is fine but not what I had in mind. We need to move on so plz fix the lint things and merge. thanks